packages/cursorless-vscode/src/ide/vscode/VscodeEnabledHatStyleManager.ts

```diff
@@ -76,9 +75,10 @@ export default class VscodeEnabledHatStyleManager {
     colorPenalties.default = 0;
 
     // So that unit tests don't fail locally if you have some colors disabled
-    const activeHatColors = isTesting()
-      ? HAT_COLORS.filter((color) => !color.startsWith("user"))
-      : HAT_COLORS.filter((color) => colorEnablement[color]);
+    const activeHatColors =
+      this.extensionContext.extensionMode === vscode.ExtensionMode.Test
```

Comment:
I think it's fine for the vscode ide to internally use extension mode. We are asserting that this one is equal to the environment variable.